app/src/main/java/com/android/unio/model/search/SearchViewModel.kt

```diff
@@ -117,6 +117,7 @@ class SearchViewModel @Inject constructor(private val repository: SearchReposito
     viewModelScope.launch {
       status.value = Status.LOADING
       val results = repository.searchEvents(query)
+      results.forEach { event -> event.organisers.requestAll() }
```

Comment:
For the same reason as the line `eventList.forEach { event -> event.organisers.requestAll() }` in the loadEvents method of EventListViewModel, it is to request the data of the associations that organise the events, so that we can then display the associations' logos and names in Home.kt and EventDetails.kt.
It's the best solution that I found, as requesting inside the components themselves made the ViewModels request continuously

Reply:
Ok seems logical, it'll probably need to be refactored at some point as I'm not sure the calls should be made here, but it works for the moment